libbeat/template/template.go

```diff
@@ -167,3 +204,19 @@ func loadYaml(path string) (Fields, error) {
 	}
 	return fields, nil
 }
+
+type Formatter struct {
```

Comment:
don't leak/export this type if you don't need it. Also `template.Formatter` is kind of a bad choice here.